ivy/functional/frontends/tensorflow/general_functions.py

```diff
@@ -316,3 +316,7 @@ def where(condition: ivy.array, x=None, y=None, name=None):
         return ivy.argwhere(condition)
     else:
         return ivy.where(condition, x, y)
+
+
+def roll(input, shift, axis, name=None):
+    return ivy.roll(input, shift, axis)
```

Comment:
axis is keyword only, see the ivy doc: 
https://lets-unify.ai/ivy/functional/ivy/manipulation/roll/roll_functional.html?highlight=*
so this should be 
```
axis=axis
```